locales/en/settings.ftl

```diff
@@ -60,3 +60,12 @@ settings-email-dialog-title = Add another email address
 settings-add-email-text = Add a new email address to see if it’s been involved in a breach.
 settings-email-input-label = Email address
 settings-send-email-verification-button = Send verification link
+
+## Unsubscribe Dialog Survey
+
+settings-unsubscribe-dialog-title = We’re sorry to see you go. <br /> Will you tell us why you’re leaving?
+settings-unsubscribe-dialog-info = Your experience is important to us. We read every response and take it into consideration.
+settings-unsubscribe-dialog-message-placeholder = What could have gone better?
+settings-unsubscribe-dialog-warning = Please note, all of your { -brand-mozilla-premium } services will be <span>permanently deleted</span> after your current billing cycle ends.
```

Comment:
```suggestion
settings-unsubscribe-dialog-warning = Please note, all of your { -brand-monitor-premium } services will  be <span>permanently deleted</span> after your current billing cycle ends.
```

Reply:
Interestingly, if I Ctrl+F this id I'm not seeing any other matches, so I'm not sure where/if it is used.